js/repl/ReplOptions.js

```diff
@@ -215,6 +217,15 @@ class ExpandedContainer extends Component<Props, State> {
               />
               File Size
             </label>
+            <label className={styles.settingsLabel}>
+            File Name
+              <input
```

Comment:
At least on the published sample, it doesn't seem like I'm actually able to type in this box?